Document: API/secondary/texttable

**********************************************
File "API/secondary/texttable.rst", line 20, in script_examples
Failed example:
    ex.script('filename.py') # Append to a Python script named 'filename.py'
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest script_examples[2]>", line 1, in <module>
        ex.script('filename.py') # Append to a Python script named 'filename.py'
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/texttable.py", line 558, in script
        fp.write("%s.color = %g\n\n" % (unique_name, self.color))
    TypeError: float argument required, not tuple
**********************************************
Document: API/graphics/meshfill

**********************************************
File "API/graphics/meshfill.rst", line 20, in script_examples
Failed example:
    ex.script('filename.py') # Append to a Python script named 'filename.py'
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest script_examples[2]>", line 1, in <module>
        ex.script('filename.py') # Append to a Python script named 'filename.py'
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/meshfill.py", line 900, in script
        fp.write("%s.missing = %g\n\n" % (unique_name, self.missing))
    TypeError: float argument required, not tuple
1 items passed all tests:
   3 tests in xyscale_meshfill
**********************************************
Document: API/graphics/vector

**********************************************
File "API/graphics/vector.rst", line 20, in script_examples
Failed example:
    ex.script('filename.py') # Append to a Python script named 'filename.py'
Expected nothing
Got:
    DEPRECATED: Use linetype or setLineAttributes instead.
**********************************************
Document: API/misc/manageElements

**********************************************
File "API/misc/manageElements.rst", line 21, in manageElements_create
Failed example:
    ex2=vcs.createprojection('projection_ex2','polar') # create 'projection_ex2' from 'polar' template
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest manageElements_create[3]>", line 1, in <module>
        ex2=vcs.createprojection('projection_ex2','polar') # create 'projection_ex2' from 'polar' template
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 120, in createprojection
        return projection.Proj(name, source)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/projection.py", line 434, in __init__
        self.type = src.type
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/projection.py", line 987, in _settype
        value = VCS_validation_functions.checkProjType(self, 'type', value)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/VCS_validation_functions.py", line 1440, in checkProjType
        checkedRaise(self, value, Exception, err)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/VCS_validation_functions.py", line 59, in checkedRaise
        raise ex(err)
    Exception: type can either be ('linear', 'utm', 'state plane', 'albers equal area', 'lambert', 'mercator', 'polar', 'polyconic', 'equid conic a', 'transverse mercator', 'stereographic', 'lambert azimuthal', 'azimuthal', 'gnomonic', 'orthographic', 'gen. vert. near per', 'sinusoidal', 'equirectangular', 'miller', 'van der grinten', 'hotin', 'robinson', 'space oblique', 'alaska', 'interrupted goode', 'mollweide', 'interrupted mollweide', 'hammer', 'wagner iv', 'wagner vii', 'oblated') or (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30)
**********************************************
File "API/misc/manageElements.rst", line 22, in manageElements_create
Failed example:
    vcs.listelements('projection') # should now contain the 'projection_ex2' projection
Expected:
    [...'projection_ex2'...]
Got:
    ['default', 'lambert', 'linear', 'mercator', 'mollweide', 'orthographic', 'polar', 'polyconic', 'projection_ex1', 'robinson']
**********************************************
File "API/misc/manageElements.rst", line 14, in manageElements_create
Failed example:
    vcs.show('textcombined') # show all available textcombined
Expected:
    *******************Textcombined Names List**********************
    ...
    *******************End Textcombined Names List**********************
Got:
    *******************Textcombined Names List**********************
    *******************End Textcombined Names List**********************
**********************************************
File "API/misc/manageElements.rst", line 18, in manageElements_create
Failed example:
    vcs.createtextcombined('EXAMPLE_tt', 'qa', 'EXAMPLE_tto', '7left') # Create 'EXAMPLE_tt' and 'EXAMPLE_tto'
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest manageElements_create[1]>", line 1, in <module>
        vcs.createtextcombined('EXAMPLE_tt', 'qa', 'EXAMPLE_tto', '7left') # Create 'EXAMPLE_tt' and 'EXAMPLE_tto'
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 1365, in createtextcombined
        Tt_name, Tt_source = check_name_source(Tt_name, Tt_source, 'texttable')
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 57, in check_name_source
        raise vcsError("Error %s object named %s already exists" % (typ, name))
    vcsError: Error texttable object named EXAMPLE_tt already exists
**********************************************
File "API/misc/manageElements.rst", line 22, in manageElements_get
Failed example:
    a.plot(ex, slab1, slab2) # plot using specified 3d_dual_scalar object
Expected:
    <vcs.displayplot.Dp ...>
Got:
    initCamera: Camera => ((0.0, 0.0, 540.0), (0.0, 0.0, 0.0), (0.0, 1.0, 0.0)) 
    <vcs.displayplot.Dp object at 0x110f707f8>
**********************************************
File "API/misc/manageElements.rst", line 21, in manageElements_get
Failed example:
    a.plot(ex, slab1) # plot using specified 3d_scalar object
Expected:
    <vcs.displayplot.Dp ...>
Got:
    initCamera: Camera => ((0.0, 0.0, 540.0), (0.0, 0.0, 0.0), (0.0, 1.0, 0.0)) 
    <vcs.displayplot.Dp object at 0x111652050>
**********************************************
File "API/misc/manageElements.rst", line 22, in manageElements_get
Failed example:
    a.plot(ex, slab1, slab2) # plot using specified 3d_vector object
Expected:
    <vcs.displayplot.Dp ...>
Got:
    Sample rate: 6 
    Sample rate: 6 
    initCamera: Camera => ((0.0, 0.0, 540.0), (0.0, 0.0, 0.0), (0.0, 1.0, 0.0)) 
    <vcs.displayplot.Dp object at 0x1116d8050>
**********************************************
File "API/misc/manageElements.rst", line 21, in manageElements_get
Failed example:
    a.taylordiagram(ex, slab1) # plot using specified taylordiagram object
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest manageElements_get[6]>", line 1, in <module>
        a.taylordiagram(ex, slab1) # plot using specified taylordiagram object
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/Canvas.py", line 1275, in taylordiagram
        return self.__plot(arglist, parms)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/Canvas.py", line 3630, in __plot
        t.plot(arglist[0], canvas=self, template=arglist[2], **keyargs)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/taylor.py", line 1965, in plot
        self.draw(canvas, data)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/taylor.py", line 1205, in draw
        d0 = float(data[i][0])
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/site-packages/numpy/ma/core.py", line 4182, in __float__
        raise TypeError("Only length-1 arrays can be converted "
    TypeError: Only length-1 arrays can be converted to Python scalars
**********************************************
File "API/misc/manageElements.rst", line 17, in manageElements_get
Failed example:
    vcs.createtextcombined('EXAMPLE_tt', 'qa', 'EXAMPLE_tto', '7left') # Create 'EXAMPLE_tt' and 'EXAMPLE_tto'
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest manageElements_get[2]>", line 1, in <module>
        vcs.createtextcombined('EXAMPLE_tt', 'qa', 'EXAMPLE_tto', '7left') # Create 'EXAMPLE_tt' and 'EXAMPLE_tto'
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 1365, in createtextcombined
        Tt_name, Tt_source = check_name_source(Tt_name, Tt_source, 'texttable')
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 57, in check_name_source
        raise vcsError("Error %s object named %s already exists" % (typ, name))
    vcsError: Error texttable object named EXAMPLE_tt already exists
1 items passed all tests:
   3 tests in manageElements_removeobject
**********************************************
Document: API/Canvas

**********************************************
File "API/Canvas.rst", line 10, in utils_getcolorcell
Failed example:
    a.getcolorcell(2,b)
Expected:
    [85, 85, 85, 100.0]
Got:
    [26, 1, 34, 100]
**********************************************
File "API/Canvas.rst", line 32, in canvas_ffmpeg
Failed example:
    a.ffmpeg('mymovie.mpeg',files="my_png__[0-9]*\.png") # generate from files with name matching regex
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest canvas_ffmpeg[9]>", line 1, in <module>
        a.ffmpeg('mymovie.mpeg',files="my_png__[0-9]*\.png") # generate from files with name matching regex
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/Canvas.py", line 4591, in ffmpeg
        if test_file is not False:
    UnboundLocalError: local variable 'test_file' referenced before assignment
**********************************************
File "API/Canvas.rst", line 18, in manageElements_create
Failed example:
    ex=vcs.create3d_dual_scalar('3d_dual_scalar_ex1') # Create 3d_dual_scalar '3d_dual_scalar_ex1' that inherits from 'default'
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest manageElements_create[1]>", line 1, in <module>
        ex=vcs.create3d_dual_scalar('3d_dual_scalar_ex1') # Create 3d_dual_scalar '3d_dual_scalar_ex1' that inherits from 'default'
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 1596, in create3d_dual_scalar
        name, source = check_name_source(name, source, '3d_dual_scalar')
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 57, in check_name_source
        raise vcsError("Error %s object named %s already exists" % (typ, name))
    vcsError: Error 3d_dual_scalar object named 3d_dual_scalar_ex1 already exists
**********************************************
File "API/Canvas.rst", line 18, in manageElements_create
Failed example:
    ex=vcs.create3d_scalar('3d_scalar_ex1') # Create 3d_scalar '3d_scalar_ex1' that inherits from 'default'
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest manageElements_create[1]>", line 1, in <module>
        ex=vcs.create3d_scalar('3d_scalar_ex1') # Create 3d_scalar '3d_scalar_ex1' that inherits from 'default'
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 1556, in create3d_scalar
        name, source = check_name_source(name, source, '3d_scalar')
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 57, in check_name_source
        raise vcsError("Error %s object named %s already exists" % (typ, name))
    vcsError: Error 3d_scalar object named 3d_scalar_ex1 already exists
**********************************************
File "API/Canvas.rst", line 18, in manageElements_create
Failed example:
    ex=vcs.create3d_vector('3d_vector_ex1') # Create 3d_vector '3d_vector_ex1' that inherits from 'default'
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest manageElements_create[1]>", line 1, in <module>
        ex=vcs.create3d_vector('3d_vector_ex1') # Create 3d_vector '3d_vector_ex1' that inherits from 'default'
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 1637, in create3d_vector
        name, source = check_name_source(name, source, '3d_vector')
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 57, in check_name_source
        raise vcsError("Error %s object named %s already exists" % (typ, name))
    vcsError: Error 3d_vector object named 3d_vector_ex1 already exists
**********************************************
File "API/Canvas.rst", line 21, in manageElements_create
Failed example:
    ex2=vcs.createprojection('projection_ex2','polar') # create 'projection_ex2' from 'polar' template
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest manageElements_create[3]>", line 1, in <module>
        ex2=vcs.createprojection('projection_ex2','polar') # create 'projection_ex2' from 'polar' template
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 120, in createprojection
        return projection.Proj(name, source)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/projection.py", line 434, in __init__
        self.type = src.type
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/projection.py", line 987, in _settype
        value = VCS_validation_functions.checkProjType(self, 'type', value)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/VCS_validation_functions.py", line 1440, in checkProjType
        checkedRaise(self, value, Exception, err)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/VCS_validation_functions.py", line 59, in checkedRaise
        raise ex(err)
    Exception: type can either be ('linear', 'utm', 'state plane', 'albers equal area', 'lambert', 'mercator', 'polar', 'polyconic', 'equid conic a', 'transverse mercator', 'stereographic', 'lambert azimuthal', 'azimuthal', 'gnomonic', 'orthographic', 'gen. vert. near per', 'sinusoidal', 'equirectangular', 'miller', 'van der grinten', 'hotin', 'robinson', 'space oblique', 'alaska', 'interrupted goode', 'mollweide', 'interrupted mollweide', 'hammer', 'wagner iv', 'wagner vii', 'oblated') or (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30)
**********************************************
File "API/Canvas.rst", line 22, in manageElements_create
Failed example:
    vcs.listelements('projection') # should now contain the 'projection_ex2' projection
Expected:
    [...'projection_ex2'...]
Got:
    ['default', 'lambert', 'linear', 'mercator', 'mollweide', 'orthographic', 'polar', 'polyconic', 'projection_ex1', 'robinson']
**********************************************
File "API/Canvas.rst", line 18, in manageElements_create
Failed example:
    vcs.createtextcombined('EXAMPLE_tt', 'qa', 'EXAMPLE_tto', '7left') # Create 'EXAMPLE_tt' and 'EXAMPLE_tto'
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest manageElements_create[1]>", line 1, in <module>
        vcs.createtextcombined('EXAMPLE_tt', 'qa', 'EXAMPLE_tto', '7left') # Create 'EXAMPLE_tt' and 'EXAMPLE_tto'
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 1365, in createtextcombined
        Tt_name, Tt_source = check_name_source(Tt_name, Tt_source, 'texttable')
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 57, in check_name_source
        raise vcsError("Error %s object named %s already exists" % (typ, name))
    vcsError: Error texttable object named EXAMPLE_tt already exists
**********************************************
File "API/Canvas.rst", line 22, in canvas_textcombined
Failed example:
    vcs.createtext('qa_tta', 'qa', '7left_tto', '7left') # Create instance of 'std_tt' and '7left_to'
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest canvas_textcombined[4]>", line 1, in <module>
        vcs.createtext('qa_tta', 'qa', '7left_tto', '7left') # Create instance of 'std_tt' and '7left_to'
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 1365, in createtextcombined
        Tt_name, Tt_source = check_name_source(Tt_name, Tt_source, 'texttable')
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 57, in check_name_source
        raise vcsError("Error %s object named %s already exists" % (typ, name))
    vcsError: Error texttable object named qa_tta already exists
**********************************************
File "API/Canvas.rst", line 22, in manageElements_get
Failed example:
    a.plot(ex, slab1, slab2) # plot using specified 3d_dual_scalar object
Expected:
    <vcs.displayplot.Dp ...>
Got:
    initCamera: Camera => ((0.0, 0.0, 540.0), (0.0, 0.0, 0.0), (0.0, 1.0, 0.0)) 
    <vcs.displayplot.Dp object at 0x1118a34b0>
**********************************************
File "API/Canvas.rst", line 21, in manageElements_get
Failed example:
    a.plot(ex, slab1) # plot using specified 3d_scalar object
Expected:
    <vcs.displayplot.Dp ...>
Got:
    initCamera: Camera => ((0.0, 0.0, 540.0), (0.0, 0.0, 0.0), (0.0, 1.0, 0.0)) 
    <vcs.displayplot.Dp object at 0x1116d3398>
**********************************************
File "API/Canvas.rst", line 22, in manageElements_get
Failed example:
    a.plot(ex, slab1, slab2) # plot using specified 3d_vector object
Expected:
    <vcs.displayplot.Dp ...>
Got:
    Sample rate: 6 
    Sample rate: 6 
    initCamera: Camera => ((0.0, 0.0, 540.0), (0.0, 0.0, 0.0), (0.0, 1.0, 0.0)) 
    <vcs.displayplot.Dp object at 0x156242910>
**********************************************
File "API/Canvas.rst", line 21, in manageElements_get
Failed example:
    a.taylordiagram(ex, slab1) # plot using specified taylordiagram object
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest manageElements_get[6]>", line 1, in <module>
        a.taylordiagram(ex, slab1) # plot using specified taylordiagram object
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/Canvas.py", line 1275, in taylordiagram
        return self.__plot(arglist, parms)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/Canvas.py", line 3630, in __plot
        t.plot(arglist[0], canvas=self, template=arglist[2], **keyargs)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/taylor.py", line 1965, in plot
        self.draw(canvas, data)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/taylor.py", line 1205, in draw
        d0 = float(data[i][0])
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/site-packages/numpy/ma/core.py", line 4182, in __float__
        raise TypeError("Only length-1 arrays can be converted "
    TypeError: Only length-1 arrays can be converted to Python scalars
**********************************************
File "API/Canvas.rst", line 17, in manageElements_get
Failed example:
    vcs.createtextcombined('EXAMPLE_tt', 'qa', 'EXAMPLE_tto', '7left') # Create 'EXAMPLE_tt' and 'EXAMPLE_tto'
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest manageElements_get[2]>", line 1, in <module>
        vcs.createtextcombined('EXAMPLE_tt', 'qa', 'EXAMPLE_tto', '7left') # Create 'EXAMPLE_tt' and 'EXAMPLE_tto'
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 1365, in createtextcombined
        Tt_name, Tt_source = check_name_source(Tt_name, Tt_source, 'texttable')
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 57, in check_name_source
        raise vcsError("Error %s object named %s already exists" % (typ, name))
    vcsError: Error texttable object named EXAMPLE_tt already exists
**********************************************
File "API/Canvas.rst", line 8, in canvas_objecthelp
Failed example:
    a.objecthelp(ln) # This will print out information on how to use ln
Expected nothing
Got:
    <BLANKLINE>
        The Line object allows the manipulation of line type, width, color index,
        view port, world coordinates, and (x,y) points.
    <BLANKLINE>
        This class is used to define an line table entry used in VCS, or it
        can be used to change some or all of the line attributes in an
        existing line table entry.
    <BLANKLINE>
        .. describe:: Useful Functions:
    <BLANKLINE>
            .. code-block:: python
    <BLANKLINE>
                # VCS Canvas Constructor
                a=vcs.init()
                # Show predefined line objects
                a.show('line')
                # Will list all the line attribute values
                ln.list()
                # Updates the VCS Canvas at user's request
                a.update()
    <BLANKLINE>
        .. describe:: Create a new instance of line:
    <BLANKLINE>
            .. code-block:: python
    <BLANKLINE>
                # Copies content of 'red' to 'new'
                ln=a.createline('new','red')
                # Copies content of 'default' to 'new'
                ln=a.createline('new')
    <BLANKLINE>
        .. describe:: Modify an existing line:
    <BLANKLINE>
            * Get a line object 'ln' to manipulate:
    <BLANKLINE>
                .. code-block:: python
    <BLANKLINE>
                    ln=a.getline('red')
    <BLANKLINE>
            * Set line color:
    <BLANKLINE>
                .. code-block:: python
    <BLANKLINE>
                    # Range from 1 to 256
                    ln.color=100
    <BLANKLINE>
            * Set line width:
    <BLANKLINE>
                .. code-block:: python
    <BLANKLINE>
                    # Range from 1 to 300
                    ln.width=100
    <BLANKLINE>
            * Specify the line type:
    <BLANKLINE>
                .. code-block:: python
    <BLANKLINE>
                    # Same as ln.type=0
                     ln.type='solid'
                     # Same as ln.type=1
                     ln.type='dash'
                     # Same as ln.type=2
                     ln.type='dot'
                     # Same as ln.type=3
                     ln.type='dash-dot'
                     # Same as ln.type=4
                     ln.type='long-dash'
    <BLANKLINE>
            * Set the graphics priority on the canvas:
    <BLANKLINE>
                .. code-block:: python
    <BLANKLINE>
                    ln.priority=1
                    # FloatType [0,1]x[0,1]
                    ln.viewport=[0, 1.0, 0,1.0]
                    # FloatType [#,#]x[#,#]
                    ln.worldcoordinate=[0,1.0,0,1.0]
    <BLANKLINE>
            * Set line x and y values:
    <BLANKLINE>
                .. code-block:: python
    <BLANKLINE>
                    # List of FloatTypes
                    ln.x=[[0,.1,.2], [.3,.4,.5]]
                    # List of FloatTypes
                    ln.y=[[.5,.4,.3], [.2,.1,0]]
    <BLANKLINE>
**********************************************
File "API/Canvas.rst", line 11, in canvas_drawtextcombined
Failed example:
    tc=a.drawtextcombined(Tt_name = 'std_example', To_name='7left_example',
                      string='Hello example!', spacing=5,
                      color=242, priority=1, viewport=[0, 1.0, 0, 1.0],
                      worldcoordinate=[0,100, 0,50],
                      x=[0,20,40,60,80,100],
                      y=[0,10,20,30,40,50]) # Create instance of texttable object 'red'
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest canvas_drawtextcombined[2]>", line 6, in <module>
        y=[0,10,20,30,40,50]) # Create instance of texttable object 'red'
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/Canvas.py", line 2314, in drawtextcombined
        t = self.gettextcombined(Tt_name, To_name)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/Canvas.py", line 2134, in gettextcombined
        x, y, height, angle, path, halign, valign)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 1482, in gettextcombined
        (Tt_name_src, To_name_src))
    Exception: No such text combined: std_example:::7left_example
**********************************************
File "API/Canvas.rst", line 17, in canvas_drawtextcombined
Failed example:
    a.textcombined(tc) # Plot using specified texttable object
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest canvas_drawtextcombined[3]>", line 1, in <module>
        a.textcombined(tc) # Plot using specified texttable object
    NameError: name 'tc' is not defined
**********************************************
File "API/Canvas.rst", line 11, in canvas_drawtextcombined
Failed example:
    tc=a.drawtextcombined(Tt_name = 'std_example', To_name='7left_example',
                      string='Hello example!', spacing=5,
                      color=242, priority=1, viewport=[0, 1.0, 0, 1.0],
                      worldcoordinate=[0,100, 0,50],
                      x=[0,20,40,60,80,100],
                      y=[0,10,20,30,40,50]) # Create instance of texttable object 'red'
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest canvas_drawtextcombined[2]>", line 6, in <module>
        y=[0,10,20,30,40,50]) # Create instance of texttable object 'red'
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/Canvas.py", line 2314, in drawtextcombined
        t = self.gettextcombined(Tt_name, To_name)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/Canvas.py", line 2134, in gettextcombined
        x, y, height, angle, path, halign, valign)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 1482, in gettextcombined
        (Tt_name_src, To_name_src))
    Exception: No such text combined: std_example:::7left_example
**********************************************
File "API/Canvas.rst", line 17, in canvas_drawtextcombined
Failed example:
    a.textcombined(tc) # Plot using specified texttable object
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest canvas_drawtextcombined[3]>", line 1, in <module>
        a.textcombined(tc) # Plot using specified texttable object
    NameError: name 'tc' is not defined
50 items passed all tests:
  14 tests in canvas_boxfill
   4 tests in canvas_clear
   4 tests in canvas_close
   4 tests in canvas_destroy
   4 tests in canvas_drawfillarea
   4 tests in canvas_drawline
   3 tests in canvas_drawlogooff
   3 tests in canvas_drawlogoon
   4 tests in canvas_drawmarker
   9 tests in canvas_eps
   9 tests in canvas_fillarea
   4 tests in canvas_flush
   4 tests in canvas_geometry
   2 tests in canvas_getcontinentstype
   4 tests in canvas_getdrawlogo
   7 tests in canvas_getfont
   6 tests in canvas_gettextextent
   4 tests in canvas_islandscape
  10 tests in canvas_isofill
   8 tests in canvas_isoline
   4 tests in canvas_isportrait
   8 tests in canvas_landscape
   9 tests in canvas_line
   2 tests in canvas_listelements
   9 tests in canvas_marker
   9 tests in canvas_meshfill
   3 tests in canvas_open
   2 tests in canvas_orientation
   6 tests in canvas_pdf
  13 tests in canvas_plot
   4 tests in canvas_png
   8 tests in canvas_portrait
   9 tests in canvas_postscript
   6 tests in canvas_pstogif
   2 tests in canvas_saveinitial_warning
  10 tests in canvas_scatter
   4 tests in canvas_scriptobject
   3 tests in canvas_setbgoutputdimensions
  11 tests in canvas_setcolorcell
   5 tests in canvas_setcolormap
   5 tests in canvas_setcontinentsline
   4 tests in canvas_setcontinentstype
   6 tests in canvas_svg
   8 tests in canvas_taylordiagram
  13 tests in canvas_update
  10 tests in canvas_vector
  11 tests in canvas_xvsy
   8 tests in canvas_xyvsy
   8 tests in canvas_yxvsx
   3 tests in manageElements_removeobject
**********************************************
Document: API/secondary/textcombined

**********************************************
File "API/secondary/textcombined.rst", line 22, in script_examples
Failed example:
    ex.script('filename.py') # Append to a Python script named 'filename.py'
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest script_examples[3]>", line 1, in <module>
        ex.script('filename.py') # Append to a Python script named 'filename.py'
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/textcombined.py", line 503, in script
        fp.write("%s.color = %g\n\n" % (unique_name, self.color))
    TypeError: float argument required, not list
**********************************************
Document: API/template/template

**********************************************
File "API/template/template.rst", line 11, in template_drawLinesAndMarkersLegend
Failed example:
    t.drawLinesAndMarkersLegend(x,
        ["red","blue","green"], ["solid","dash","dot"],[1,4,8],
        ["blue","green","red"], ["cross","square","dot"],[3,4,5],
        ["sample A","type B","thing C"],True)
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest template_drawLinesAndMarkersLegend[3]>", line 4, in <module>
        ["sample A","type B","thing C"],True)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/template.py", line 1522, in drawLinesAndMarkersLegend
        strings, scratched, bg, render)
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/utils.py", line 2388, in drawLinesAndMarkersLegend
        if scratched is not None and scratched[i] is not False:
    TypeError: 'bool' object has no attribute '__getitem__'
5 items passed all tests:
   4 tests in script_examples
   3 tests in template_move
   2 tests in template_moveto
   2 tests in template_reset
   4 tests in template_scale
**********************************************
Document: API/misc/vcshelp

**********************************************
File "API/misc/vcshelp.rst", line 8, in vcshelp_objecthelp
Failed example:
    for object in objects:
        vcs.objecthelp(object)
Expected nothing
Got:
    None
    <BLANKLINE>
        The Colormap object allows the manipulation of the colormap index R,G,B values.
    <BLANKLINE>
        This class is used to define a colormap table entry used in VCS, or it
        can be used to change some or all of the colormap R,G,B attributes in an
        existing colormap table entry.
    <BLANKLINE>
        .. describe:: Some Useful Functions:
    <BLANKLINE>
            .. code-block:: python
    <BLANKLINE>
                # Constructor
                a=vcs.init()
                # Show predefined colormap objects
                a.show('colormap')
                # Updates the VCS Canvas at user's request
                a.update()
                # If mode=1, automatic update
                a.mode=1
                #If mode=0, use update function to update the VCS Canvas.
                a.mode=0
    <BLANKLINE>
    <BLANKLINE>
    <BLANKLINE>
        .. describe:: General use of a colormap:
    <BLANKLINE>
            .. code-block:: python
    <BLANKLINE>
                # Create a VCS Canvas object
                a=vcs.init()
                #To Create a new instance of colormap use:
                # Copies content of 'red' to 'new'
                cp=a.createcolormap('new','quick')
                # Copies content of 'default' to 'new'
                cp=a.createcolormap('new')
    <BLANKLINE>
        .. describe:: Modifying an existing colormap:
    <BLANKLINE>
            .. code-block:: python
    <BLANKLINE>
                cp=a.getcolormap('quick')
    <BLANKLINE>
        .. describe:: Overview of colormap attributes:
    <BLANKLINE>
            * List all the colormap indices and R,G,B attribute values
    <BLANKLINE>
                .. code-block:: python
    <BLANKLINE>
                    cp.list()
    <BLANKLINE>
            * Setting colormap attribute values:
    <BLANKLINE>
                .. code-block:: python
    <BLANKLINE>
                    # Index, R, G, B
                    cp.color=16,100,0,0
                    # Index range from 0 to 255, but can only modify from 0 to 239
                    cp.color=16,0,100,0
                    # R, G, B values range from 0 to 100, where 0 is low intensity and 100 is highest intensity
                    cp.color=17,0,0,100
    <BLANKLINE>
    <BLANKLINE>
    <BLANKLINE>
        The boxfill graphics method (Gfb) displays a two-dimensional data array
        by surrounding each data value by a colored grid box.
    <BLANKLINE>
        This class is used to define a boxfill table entry used in VCS, or it
        can be used to change some or all of the attributes in an existing
        boxfill table entry.
    <BLANKLINE>
    <BLANKLINE>
        .. describe:: General use of a boxfill:
    <BLANKLINE>
                .. code-block:: python
    <BLANKLINE>
                    # Constructor
                    a=vcs.init()
                    # Show predefined boxfill graphics methods
                    a.show('boxfill')
                    # Change the VCS color map
                    a.setcolormap("AMIP")
                    # Plot data 's' with boxfill 'b' and 'default' template
                    a.boxfill(s,b,'default')
    <BLANKLINE>
        .. describe:: Updating a boxfill:
    <BLANKLINE>
            .. code-block:: python
    <BLANKLINE>
                # Updates the VCS Canvas at user's request
                a.update()
                # Set VCS Canvas to automatic update mode
                a.mode=1
                # Use update function to update the VCS Canvas
                a.mode=0
    <BLANKLINE>
        .. describe:: Create a new instance of boxfill:
    <BLANKLINE>
            .. code-block:: python
    <BLANKLINE>
                #  Copies content of 'quick' to 'new'
                box=a.createboxfill('new','quick')
                #  Copies content of 'default' to 'new'
                box=a.createboxfill('new')
    <BLANKLINE>
        .. describe:: Modifying an existing boxfill:
    <BLANKLINE>
            .. code-block:: python
    <BLANKLINE>
                fill=a.getboxfill('quick')
    <BLANKLINE>
                # Set index using fillarea
                box.fillareaindices=(7,fill,4,9,fill,15)
                # list fillarea attributes
                fill.list()
                # change style
                fill.style='hatch'
                # change color
                fill.color='black'
                # change style index
                fill.index=3
    <BLANKLINE>
        .. describe:: Overview of boxfill attributes:
    <BLANKLINE>
            * Listing all the boxfill attribute values:
    <BLANKLINE>
                .. code-block:: python
    <BLANKLINE>
                    box.list()
    <BLANKLINE>
            * Setting boxfill attribute values:
    <BLANKLINE>
                .. code-block:: python
    <BLANKLINE>
                    box.projection='linear'
                    lon30={-180:'180W',-150:'150W',0:'Eq'}
                    box.xticlabels1=lon30
                    box.xticlabels2=lon30
                    # Will set them both
                    box.xticlabels(lon30, lon30)
                    box.xmtics1=''
                    box.xmtics2=''
                    # Will set them both
                    box.xmtics(lon30, lon30)
                    box.yticlabels1=lat10
                    box.yticlabels2=lat10
                    # Will set them both
                    box.yticlabels(lat10, lat10)
                    box.ymtics1=''
                    box.ymtics2=''
                    # Will set them both
                    box.ymtics(lat10, lat10)
                    box.datawc_y1=-90.0
                    box.datawc_y2=90.0
                    box.datawc_x1=-180.0
                    box.datawc_x2=180.0
                    # Will set them all
                    box.datawc(-90, 90, -180, 180)
                    box.xaxisconvert='linear'
                    box.yaxisconvert='linear'
                    # Will set them both
                    box.xyscale('linear', 'area_wt')
                    box.level_1=1e20
                    box.level_2=1e20
                    box.color_1=0
                    box.color_2=255
                    # Will set them both
                    box.colors(0, 255 )
                    # 'linear' - compute or specify legend
                    box.boxfill_type='linear'
                    # 'log10' - plot using log10
                    box.boxfill_type='log10'
                    # 'custom' - use custom values to display legend evenly
                    box.boxfill_type='custom'
                    # Hold the legend values
                    box.legend=None
                    # Show left overflow arrow
                    box.ext_1='n'
                    # Show right overflow arrow
                    box.ext_2='y'
                    # Will set them both
                    box.exts('n', 'y' )
                    box.missing='black'
    <BLANKLINE>
            * Setting the boxfill levels:
    <BLANKLINE>
                .. code-block:: python
    <BLANKLINE>
                    # Case 1: Levels are all contiguous:
                    box.levels=([0,20,25,30,35,40],)
                    box.levels=([0,20,25,30,35,40,45,50])
                    box.levels=[0,20,25,30,35,40]
                    box.levels=(0.0,20.0,25.0,30.0,35.0,40.0,50.0)
    <BLANKLINE>
                    # Case 2: Levels are not contiguous:
                    box.levels=([0,20],[30,40],[50,60])
                    box.levels=([0,20,25,30,35,40],[30,40],[50,60])
    <BLANKLINE>
            * Setting the fillarea color indices:
    <BLANKLINE>
                .. code-block:: python
    <BLANKLINE>
                    # Three different methods for setting color indices:
                    box.fillareacolors=([22,33,44,55,66,77])
                    box.fillareacolors=(16,19,33,44)
                    box.fillareacolors=None
    <BLANKLINE>
            * Setting the fillarea style:
    <BLANKLINE>
                .. code-block:: python
    <BLANKLINE>
                    box.fillareastyle = 'solid'
                    box.fillareastyle = 'hatch'
                    box.fillareastyle = 'pattern'
    <BLANKLINE>
            * Setting the fillarea hatch or pattern indices:
    <BLANKLINE>
                .. code-block:: python
    <BLANKLINE>
                    box.fillareaindices=([1,3,5,6,9,20])
                    box.fillareaindices=(7,1,4,9,6,15)
    <BLANKLINE>
            * Using the fillarea secondary object (Ex):
    <BLANKLINE>
                .. code-block:: python
    <BLANKLINE>
                    f=createfillarea('fill1')
                    #To Create a new instance of fillarea use:
                    # Copies 'quick' to 'new'
                    fill=a.createfillarea('new','quick')
                    # Copies 'default' to 'new'
                    fill=a.createfillarea('new')
    <BLANKLINE>
            .. _boxfill-attribute-descriptions:
    <BLANKLINE>
            * Attribute descriptions:
    <BLANKLINE>
    <BLANKLINE>
                .. py:attribute:: boxfill_type (str)
    <BLANKLINE>
                    Type of boxfill legend. One of 'linear', 'log10', or 'custom'. See examples above for usage.
    <BLANKLINE>
                .. py:attribute:: level_1 (float)
    <BLANKLINE>
                    Used in conjunction with boxfill_type linear/log10. Sets the value of the legend's first level
    <BLANKLINE>
                .. py:attribute:: level_2 (float)
    <BLANKLINE>
                    Used in conjunction with boxfill_type linear/log10, sets the value of the legend's end level
    <BLANKLINE>
                .. py:attribute:: color_1 (float)
    <BLANKLINE>
                    Used in conjunction with boxfill_type linear/log10, sets the legend's color range first value
    <BLANKLINE>
                .. py:attribute:: color_2 (float)
    <BLANKLINE>
                    Used in conjunction with boxfill_type linear/log10, sets the legend's color range lasst value
    <BLANKLINE>
                .. py:attribute:: levels (list of floats)
    <BLANKLINE>
                    Used in conjunction for boxfill_type custom, sets the levels range to use, can be
                    either a list of contiguous levels, or list of tuples indicating first
                    and last value of the range.
    <BLANKLINE>
                .. py:attribte:: fillareacolors (list)
    <BLANKLINE>
                    Used in conjunction for boxfill_type custom colors to use for each level
    <BLANKLINE>
                .. py:attribute:: legend ({float:str})
    <BLANKLINE>
                    Used in conjunction with boxfill_type linear/log10, replaces the
                    legend values in the dictionary keys with their associated string.
    <BLANKLINE>
                .. py:attribute:: ext_1 (str)
    <BLANKLINE>
                    Draws an extension arrow on right side (values less than first range value)
    <BLANKLINE>
                .. py:attribute:: ext_2 (str)
    <BLANKLINE>
                    Draws an extension arrow on left side (values greater than last range value)
    <BLANKLINE>
                .. py:attribute:: missing (int)
    <BLANKLINE>
                    Color to use for missing value or values not in defined ranges.
    <BLANKLINE>
    <BLANKLINE>
    <BLANKLINE>
                .. py:attribute:: xmtics1 (str/{float:str})
    <BLANKLINE>
                    (Ex: '') dictionary with location of intermediate tics as keys for 1st side of y axis
    <BLANKLINE>
                .. py:attribute:: xmtics2 (str/{float:str})
    <BLANKLINE>
                    (Ex: '') dictionary with location of intermediate tics as keys for 2nd side of y axis
    <BLANKLINE>
                .. py:attribute:: ymtics1 (str/{float:str})
    <BLANKLINE>
                    (Ex: '') dictionary with location of intermediate tics as keys for 1st side of y axis
    <BLANKLINE>
                .. py:attribute:: ymtics2 (str/{float:str})
    <BLANKLINE>
                    (Ex: '') dictionary with location of intermediate tics as keys for 2nd side of y axis
    <BLANKLINE>
                .. py:attribute:: xticlabels1 (str/{float:str})
    <BLANKLINE>
                    (Ex: '*') values for labels on 1st side of x axis
    <BLANKLINE>
                .. py:attribute:: xticlabels2 (str/{float:str})
    <BLANKLINE>
                    (Ex: '*') values for labels on 2nd side of x axis
    <BLANKLINE>
                .. py:attribute:: yticlabels1 (str/{float:str})
    <BLANKLINE>
                    (Ex: '*') values for labels on 1st side of y axis
    <BLANKLINE>
                .. py:attribute:: yticlabels2 (str/{float:str})
    <BLANKLINE>
                    (Ex: '*') values for labels on 2nd side of y axis
    <BLANKLINE>
                .. py:attribute:: projection (str/vcs.projection.Proj)
    <BLANKLINE>
                    (Ex: 'default') projection to use, name or object
    <BLANKLINE>
                .. py:attribute:: datawc_x1 (float)
    <BLANKLINE>
                    (Ex: 1.E20) first value of xaxis on plot
    <BLANKLINE>
                .. py:attribute:: datawc_x2 (float)
    <BLANKLINE>
                    (Ex: 1.E20) second value of xaxis on plot
    <BLANKLINE>
                .. py:attribute:: datawc_y1 (float)
    <BLANKLINE>
                    (Ex: 1.E20) first value of yaxis on plot
    <BLANKLINE>
                .. py:attribute:: datawc_y2 (float)
    <BLANKLINE>
                    (Ex: 1.E20) second value of yaxis on plot
    <BLANKLINE>
    <BLANKLINE>
                .. py:attribute:: datawc_timeunits (str)
    <BLANKLINE>
                    (Ex: 'days since 2000') units to use when displaying time dimension auto tick
    <BLANKLINE>
                .. py:attribute:: datawc_calendar (int)
    <BLANKLINE>
                    (Ex: 135441) calendar to use when displaying time dimension auto tick, default is proleptic gregorian calendar
    <BLANKLINE>
    <BLANKLINE>
    <BLANKLINE>
    <BLANKLINE>
**********************************************
Document: API/misc/queries

**********************************************
File "API/misc/queries.rst", line 11, in queries_isplot
Failed example:
    vcs.queries.isplot(dsp_plot)
Expected:
    1
Got:
    0
**********************************************
File "API/misc/queries.rst", line 7, in queries_is
Failed example:
    vcs.createtextcombined('EXAMPLE_tt', 'qa', 'EXAMPLE_tto', '7left') # Create 'EXAMPLE_tt' and 'EXAMPLE_tto'
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest queries_is[1]>", line 1, in <module>
        vcs.createtextcombined('EXAMPLE_tt', 'qa', 'EXAMPLE_tto', '7left') # Create 'EXAMPLE_tt' and 'EXAMPLE_tto'
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 1365, in createtextcombined
        Tt_name, Tt_source = check_name_source(Tt_name, Tt_source, 'texttable')
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/manageElements.py", line 57, in check_name_source
        raise vcsError("Error %s object named %s already exists" % (typ, name))
    vcsError: Error texttable object named EXAMPLE_tt already exists
4 items passed all tests:
   2 tests in queries_gmlist
   7 tests in queries_gmtype
   3 tests in queries_isgraphicsmethod
   4 tests in queries_issecondary
**********************************************
Document: API/graphics/isofill

**********************************************
File "API/graphics/isofill.rst", line 20, in script_examples
Failed example:
    ex.script('filename.py') # Append to a Python script named 'filename.py'
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest script_examples[2]>", line 1, in <module>
        ex.script('filename.py') # Append to a Python script named 'filename.py'
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/isofill.py", line 856, in script
        fp.write("%s.missing = %g\n" % (unique_name, self.missing))
    TypeError: float argument required, not tuple
1 items passed all tests:
   3 tests in xyscale_isofill
**********************************************
Document: API/misc/utils

**********************************************
File "API/misc/utils.rst", line 10, in utils_getcolorcell
Failed example:
    a.getcolorcell(2,b)
Expected:
    [85, 85, 85, 100.0]
Got:
    [26, 1, 34, 100]
**********************************************
File "API/misc/utils.rst", line 8, in utils_mklabels
Failed example:
    vcs.mklabels (a)
Expected:
    {20.0: '20', 18.0: '18', 16.0: '16', 14.0: '14', 12.0: '12',
        10.0: '10', 8.0: '8', 6.0: '6', 4.0: '4', 2.0: '2', 0.0: '0'}
Got:
    {0.0: '0', 2.0: '2', 4.0: '4', 6.0: '6', 8.0: '8', 10.0: '10', 12.0: '12', 14.0: '14', 16.0: '16', 18.0: '18', 20.0: '20'}
**********************************************
File "API/misc/utils.rst", line 11, in utils_mklabels
Failed example:
    vcs.mklabels ( [5,.005])
Expected:
    {0.0050000000000000001: '0.005', 5.0: '5.000'}
Got:
    {5.0: '5.000', 0.005: '0.005'}
**********************************************
File "API/misc/utils.rst", line 13, in utils_mklabels
Failed example:
    vcs.mklabels ( [.00002,.00005])
Expected:
    {2.0000000000000002e-05: '2E-5', 5.0000000000000002e-05: '5E-5'}
Got:
    {2e-05: '2E-5', 5e-05: '5E-5'}
**********************************************
File "API/misc/utils.rst", line 8, in utils_match_color
Failed example:
    print vcs.match_color('salmon')
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest utils_match_color[1]>", line 1, in <module>
        print vcs.match_color('salmon')
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/utils.py", line 1958, in match_color
        colormap = vcs.getcolormapname()
    AttributeError: 'module' object has no attribute 'getcolormapname'
**********************************************
File "API/misc/utils.rst", line 9, in utils_match_color
Failed example:
    print vcs.match_color('red')
Exception raised:
    Traceback (most recent call last):
      File "/Users/brown308/anaconda/envs/2.8/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest utils_match_color[2]>", line 1, in <module>
        print vcs.match_color('red')
      File "/Users/brown308/projects/embrown/documentation/vcs/vcs/vcs/utils.py", line 1958, in match_color
        colormap = vcs.getcolormapname()
    AttributeError: 'module' object has no attribute 'getcolormapname'
**********************************************
File "API/misc/utils.rst", line 10, in utils_match_color
Failed example:
    print vcs.match_color([0,0,100],'default') # closest color from blue
Expected nothing
Got:
    52
12 items passed all tests:
  13 tests in utils_Logo
  13 tests in utils_Logo_plot
   5 tests in utils_drawLinesAndMarkersLegend
   4 tests in utils_generate_time_labels
   4 tests in utils_getcolormap
   7 tests in utils_getcolors
   4 tests in utils_minmax
   3 tests in utils_mkevenlevels
   6 tests in utils_mkscale
   2 tests in utils_rbga_color
   6 tests in utils_setcolorcell
   6 tests in utils_show
**********************************************
